docs/05-concepts/06-database/01-connection.md

```diff
@@ -62,3 +70,59 @@ To remove the database and __delete__ all data add the `-v` flag.
 $ docker compose down -v
 ```
 
+
+## Connecting to a Postgresql Server Instance
+
+Connecting to a Postgresql Server Instance is similar to connecting to a Docker Image, but there are some prerequisites to take into account:
+
+- Making sure that the machine/server allows incoming traffic
+- Creating a Postgresql user and database
+
+You can connect to Postgresql server that you have installed on your local machine, a self-hosted one, Google Cloud SQL, AWS RDS or SQL Server, or any other 3rd party service.
+
+### Connecting to a Local Postgresql
+
+If you want to connect to a local Postgresql Server (with the default setup) then the `development.yaml` will work just fine. The only thing you need to do is change the password in the `passwords.yaml`.
+
+### Connecting to a Remote Postgresql
+
+To connect to a remote Postgresql Server (that you have installed on a VPS or VDS) you need to follow a couple of steps:
+- Make sure that the Postgresql Server has a reachable IP address (that it accepts incoming traffic)
+- Make sure to open any necessary ports on that machine
+- Configure Postgresql Server to listen to all other connections (not only local once) in `postgresql.conf`:
+```bash
+listen_addresses = '*'
+```
+
+- Edit your Postgresql Server `pg_hba.conf` to accept incoming traffic (this config accepts all incoming traffic which is not recommended):
+
+```bash
+host    all             all              0.0.0.0/0                       md5
+host    all             all              ::/0                            md5
+```
+
+- Update your Serverpod `database` config to use the public IP, database name, and user. Also the database password in `passwords.yaml`.
+
+
+### Connecting to Google Cloud SQL
+
+You can connect to a Google Cloud SQL Postgresql Instance in 2 ways:
+
+- Setting up the Public IP Authorized networks (with your Serverpod server IP) and changing the database host string to the `Cloud SQL public IP`
+- Using the Connection String if you are hosting your Serverpod server on Google Cloud Run and changing the database host string to the Cloud SQL: `/cloudsql/my-project:server-location:database-name/.s.PGSQL.5432`
```

Comment:
Question: Most parts of the link that should be used are configured by the user. Does it make sense to clarify that these are user-provided as opposed to the constant parts such as `.s.PGSQL.5432`?

We could provide a small explanation on the user provided parts as well.

```suggestion
- Using the Connection String if you are hosting your Serverpod server on Google Cloud Run and changing the database host string to the Cloud SQL: `/cloudsql/<my-project>:<server-location>:<database-name>/.s.PGSQL.5432`
```

Reply:
Same question for the AWS RDS link.